content/en/mobile/_index.md

```diff
@@ -73,210 +73,161 @@ To log out, navigate to the **Settings** page on the mobile app and click on **L
-
+{{< img src="service_management/mobile/Android_Notebooks.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Android notebooks page showing list of notebooks with search and filter options">}}
 
 {{% /tab %}}
 {{< /tabs >}}
 
+On the Notebooks page, you can view and search all of the notebooks that you have access to in your Datadog org, and filter them by tags. Notebook tags allow you to filter by favorites, team, and type. See [notebook tags][19] for more information.
+
+**Note**: To set up or edit a notebook, you need to [log in to the Datadog browser app][10]. For more information, see [Notebooks][18].
+
 ## Traces
 
 {{< tabs >}}
 {{% tab "iOS" %}}
 
-{{< img src="service_management/mobile/trace_may_2025.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Traces page">}}
-
-On the Traces page, you can view and search all of the traces that you have access to in your Datadog org. You can narrow the list through saved views or build specific search queries based on your tagging strategy. For more information about search, see [Trace Explorer Query Syntax][16].
-
-For example, to filter on traces with the tag `#env:prod` or the tag `#test`, use the query `"env:prod" OR test`. Click into individual services to expand associated spans, and select spans to view info, errors, and related logs. You can also open traces from services and logs.
-
-Additionally, Watchdog Insights surface latency outliers and error outliers. For more information, see [Watchdog Insights][26].
-**Note:** Watchdog is only available on iOS.
+{{< img src="service_management/mobile/trace_may_2025.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="iOS traces page showing list of traces with search and filter options">}}
 
 {{% /tab %}}
-
 {{% tab "Android" %}}
 
-{{< img src="service_management/mobile/Android_Traces.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Traces page">}}
+{{< img src="service_management/mobile/Android_Traces.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Android traces page showing list of traces with search and filter options">}}
+
+{{% /tab %}}
+{{< /tabs >}}
 
 On the Traces page, you can view and search all of the traces that you have access to in your Datadog org. You can narrow the list through saved views or build specific search queries based on your tagging strategy. For more information about search, see [Trace Explorer Query Syntax][16].
 
 For example, to filter on traces with the tag `#env:prod` or the tag `#test`, use the query `"env:prod" OR test`. Click into individual services to expand associated spans, and select spans to view info, errors, and related logs. You can also open traces from services and logs.
 
-{{% /tab %}}
-{{< /tabs >}}
+**Only available on iOS** Watchdog Insights surface latency outliers and error outliers. For more information, see [Watchdog Insights][26].
```

Comment:
```suggestion
**Only available on iOS**: Watchdog Insights point to latency outliers and error outliers. For more information, see [Watchdog Insights][26].
```

i feel "surface" is a weird verb that most people are likely to interpret as a noun at first